resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -757,11 +757,14 @@ sqs.message.validation.empty.deduplication_id=Enter a deduplication ID.
 sqs.message.validation.empty.group_id=Enter a group ID.
 sqs.message.validation.empty.message.body=Enter a message body
 sqs.message.validation.long.id=ID exceeds the maximum length of 128 characters.
-sqs.message_table_initial_text=Press "Poll for Messages" to load messages
+sqs.message_table_initial_text=Press "Peek Messages" to load messages
 sqs.messages.available.text={0, choice, 0#No messages|1#1 message|2#{0,number} messages} available
-sqs.poll.message=Poll for Messages
-sqs.poll.warning.text=<html><p>Once polled, the sample of messages remain in the queue and are not returned to subsequent receive requests for the duration of the visibility timeout.</p></html>
-sqs.purge_queue=Purge Queue
+sqs.poll.message=Peek Messages
+sqs.poll.warning.text=<html><p>Messages are returned to the queue immediately upon peeking.</p></html>
+sqs.purge_queue=Purge Queue...
+sqs.purge_queue.action=Purge Queue
+sqs.purge_queue.confirm=Are you sure you want to purge queue {0}?
```

Comment:
Should we add an approximate message count?

"Are you sure you want to purge queue 'myQueue' (approx. 23,234,234 messages)?"